integration/util/waysnodes/nodes2wayblotdb/db.go

```diff
@@ -0,0 +1,214 @@
+// Package nodes2wayblotdb stores `fromNodeID,toNodeID -> wayID` mapping in blotdb.
+package nodes2wayblotdb
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/golang/glog"
+
+	bolt "go.etcd.io/bbolt"
+)
+
+// DB stores `fromNodeID,toNodeID -> wayID` in blotdb.
+type DB struct {
+	db *bolt.DB
+}
+
+const (
+	defaultBucket = "bucket"
```

Comment:
Shall we give bucket a specific name, like `node2wayids_bucket`?  Will it possible that we have same db but would have different buckets for queries?

Reply:
Maybe it's good to have a named bucket even we may only has one. I'll rename it.